AnagramValidityChecker.java

```diff
@@ -0,0 +1,83 @@
+package CodeUAss1Package;
+
+import java.util.HashMap;
+
+public class AnagramValidityChecker
+{
+    /* Given two strings, determine if one is an anagram of the other.
+   Two sentences are anagrams if each word in the resulting
+    sentence is an anagram of one of the words in the original sentence.*/
+    public static boolean isSentenceAnagram(String sentence1, String sentence2, boolean isCaseSensitive)
+    {
+        // split the string to the words they are containing
+        String[] wordsOfSentence1 = sentence1.split(" ");
+        String[] wordsOfSentence2 = sentence2.split(" ");
+
+        if(wordsOfSentence1.length != wordsOfSentence2.length) return false;
+
+        boolean[] isSent2WordAnagramFound = new boolean[wordsOfSentence2.length];
+
+        for(int i = 0 ; i < wordsOfSentence1.length ; i++)
+        {
+            for(int j = 0 ; j < wordsOfSentence2.length ; j++)
+            {
+                // if current word has already found in the other sentence than
+                // we should'nt check it again.
+                if (isSent2WordAnagramFound[j]) continue;
+
+                // checks if the current word is anagram of the other, and mark it in isSent2WordAnagramFound in case one is the anagram of the other.

```

Comment:
Please break lines at some reasonable length. In this case new line before `isSent2WordAnagramFound` would be perfect. Shorter lines are easier to read.